modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/extensions/Modifier.kt

```diff
@@ -86,3 +94,37 @@ enum class FadeDirection {
     TopToBottom,
     BottomToTop,
 }
+
+fun Modifier.verticalScrollBar(
+    scrollState: ScrollState,
+    viewPortHeight: Float,
+    width: Dp = 4.dp,
+    thumbColor: Color,
+) = composed {
+    val thumbAlphaAnimated by animateFloatAsState(
+        targetValue = if (scrollState.isScrollInProgress) 0.8f else 0f,
+        animationSpec = tween(
+            durationMillis = if (scrollState.isScrollInProgress) 150 else 1000,
+            delayMillis = if (scrollState.isScrollInProgress) 0 else 500,
+        ),
+        label = "",
+    )
+
+    drawWithContent {
+        drawContent()
+        if (scrollState.maxValue == 0) {
+            return@drawWithContent
+        }
+        val contentHeight = size.height
+        val scrollBarHeight = viewPortHeight * viewPortHeight / contentHeight
```

Comment:
```suggestion
    val miScrollBarHeight = LocalDensity.current.run { 16.dp.toPx() }

    drawWithContent {
        drawContent()
        if (scrollState.maxValue == 0) {
            return@drawWithContent
        }
        val contentHeight = size.height
        val scrollBarHeight = (viewPortHeight * viewPortHeight / contentHeight).coerceAtLeast(miScrollBarHeight)
```

Reply:
Applied in [55fd90d](https://github.com/Automattic/pocket-casts-android/pull/2556/commits/55fd90da04772cd004d327687ea0b289c59ea893)